packages/components/src/data-model/constants/error-severity-types.js

```diff
@@ -0,0 +1,9 @@
+/*
+All possible severity values that can be returned from the GraphQL API.
+ */
+/** @enum {string} */
+const ErrorSeverityTypes = {
+  ERROR: 'Error',
+  WARNING: 'Warning'
+}
+export default ErrorSeverityTypes
```

Comment:
This is a way to distinguish between an error and a warning within the GraphQL `error` object (as GraphQL specs do not support a `warning` type).